android/wallet-core/maven-push.gradle

```diff
@@ -1,33 +1,10 @@
 apply plugin: 'maven-publish'
 
-task sourcesJar(type: Jar) {
-    archiveClassifier.set('sources')
-    from android.sourceSets.main.java.sourceFiles
-}
-
-artifacts {
-    archives sourcesJar
-}
-
 publishing {
     publications {
-        Production(MavenPublication) {
-            artifact("$buildDir/outputs/aar/wallet-core-release.aar")
-            artifact sourcesJar {
-                classifier "sources"
-            }
-
-            pom.withXml {
-                def dependenciesNode = asNode().appendNode('dependencies')
-
-                configurations.implementation.allDependencies.each {
-                    if (it.name != 'unspecified') {
-                        def dependencyNode = dependenciesNode.appendNode('dependency')
-                        dependencyNode.appendNode('groupId', it.group)
-                        dependencyNode.appendNode('artifactId', it.name)
-                        dependencyNode.appendNode('version', it.version)
-                    }
-                }
+        release(MavenPublication) {
+            afterEvaluate {
```

Comment:
In examples, they use `afterEvaluate` for some reason
https://developer.android.com/build/publish-library/upload-library#create-pub